traits/WhereTrait.php

```diff
@@ -36,17 +36,39 @@ private function initWhereAnd():void {
             }
         }
 
+        /**
+         * Initiates string starter for first call of 'whereAnd' method
+         *
+         * @return void
+         */
```

Comment:
Added docs because they were missing for some reason; this method should have been documented a long time ago.